src/components/Main/Navbar.tsx

```diff
@@ -59,7 +60,9 @@ const Navbar = () => {
 		console.log('click ', e)
 		setCurrent(e.key)
 	}
-	const authTokenJSON = JSON.parse(localStorage.getItem('userData'))
+	const authTokenJSON = JSON.parse(
+		localStorage.getItem(LOCAL_STORAGE.SESSION_INFO)
```

Comment:
@smeet1406 
We can further improve this by replacing current line with below code -
```suggestion
 const authTokenJSON = safeLocalStorage.get<LoginUserRes>(LOCAL_STORAGE.SESSION_INFO);
```